### PR TITLE
privateGPT example in DockerFile + docker-compose.yml

### PR DESCRIPTION
**Pull request by womd**
Example of using PrivateGPT on LocalAI

- currently there are some troubles setting up PrivateGPT in this example, mainly because of specific versions of python packages used. This shall provide an "easy to fire up example" to get things going.

### CODE REVIEW

File: examples/privateGPT/README.md [hunk 1]
```diff
@@ -1,25 +1,52 @@
...
+### query
+
+python privateGPT.py -S -M
 
 ```bash
 git clone git@github.com:go-skynet/privateGPT.git
-cd privateGPT
-pip install -r requirements.txt
-```
+cp example.env ./privateGTP/.env
```
Comment (mudler) [inline examples/privateGPT/README.md:22]:
```suggestion
cp example.env ./privateGPT/.env
```

File: examples/privateGPT/entrypoint.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+tail -f /dev/null
```
Comment (mudler) [inline examples/privateGPT/entrypoint.sh:3]: is this a leftover?

File: examples/privateGPT/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+WORKDIR /app/privateGPT
+
+
+COPY ./privateGPT/requirements.txt .
+
+RUN pip install -r requirements.txt
+
+
+CMD ["tail", "-f", "/dev/null"]
```
Comment (mudler) [inline examples/privateGPT/Dockerfile:21]: another leftover? shouldn't use the entrypoint?

File: examples/privateGPT/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+RUN python3 -m ensurepip
+RUN pip3 install --no-cache --upgrade pip setuptools
+RUN pip install python-dotenv tqdm langchain
+RUN pip install chromadb==0.3.29 --no-deps
+
+
+WORKDIR /app
+
+#RUN git clone https://github.com/go-skynet/privateGPT
```
Comment (mudler) [inline examples/privateGPT/Dockerfile:11]: should this clone privateGPT eventually, right?

File: examples/privateGPT/Dockerfile (hunk 3 above)
Comment (womd) [reply examples/privateGPT/Dockerfile:21]: yes, they were leftovers, the container would just stay up indefinetly, one could connect to it via docker exec -it then run the python scrpts. Tought about having a simple rest-endpoint to query it, but not sure, intention was just to get it running. How would you like to have it running ?

Comment (womd) [reply examples/privateGPT/Dockerfile:20]: updated, please tell me what you think now

**Pull request closed without merging**